tests/testthat/test_measure.R

```diff
@@ -0,0 +1,53 @@
+
+
+test_that("measure_test1", {
+  config <- create.config(df = example.data, id = "sweden_at_eurovision_no_missing") %>%
+    add.measure.column(
+      column.name = "Measure",
+      values = values(
+        measure.value(label = "Final Rank", description = "jfeaiji", definition.uri = "https://www.amazon.co.uk/s?k=apple+gift+card+—+app+store%2C+itunes%2C+iphone%2C+ipad%2C+airpods%2C+macbook%2C+accessories+and+more+%28email+delivery%29+–+for+uk&crid=AMQTV99P1F42&sprefix=%2Caps%2C51&ref=nb_sb_ss_recent_1_0_recent", from.existing = measure.value.from.existing$`travel-time`),
```

Comment:
Again can you remove these Amazon URLs?

If you want some test URL, you can use `http://example.com/....` as your basis for making things up.